website-components-playground/src/pages/tabular-modal-page.js

```diff
@@ -50,105 +40,131 @@ const exampleCustomTitleRow = (
-                  </span>
-                </TabControlsContainer>
-              }
-              customTitleRow={
-                values.useCustomTitleRow === 'custom' && exampleCustomTitleRow
-              }
-              formControls={
-                <>
-                  <TabularModalPage.FormSecondaryButton
-                    onClick={() => {
-                      setIsOpen(false);
-                    }}
-                  />
-                  <TabularModalPage.FormPrimaryButton
-                    onClick={() => undefined}
-                    isDisabled
-                  />
-                  <TabularModalPage.FormDeleteButton
-                    onClick={() => undefined}
-                    isDisabled
-                  />
-                </>
-              }
-              hideControls={values.hideControls}
+              buttonLabel="Open Tabular Modal Page"
+              containerId={containerId}
             >
-              <Text.Body>{values.content}</Text.Body>
-            </TabularModalPage>
+              {({ isOpen, setIsOpen }) => (
+                <TabularModalPage
+                  title={values.title}
+                  isOpen={isOpen}
+                  onClose={() => setIsOpen(false)}
+                  getParentSelector={() =>
+                    document.querySelector(`#${containerId}`)
+                  }
+                  tabControls={
+                    <>
+                      <TabHeader name="tab-one" to="/tab-one">
+                        Tab One
+                      </TabHeader>
+                      <TabHeader to="/tab-two">Tab Two</TabHeader>
+                      <TabHeader name="tab-three" to="/tab-three" isDisabled>
+                        Disabled tab
+                      </TabHeader>
+                    </>
+                  }
+                  customTitleRow={
+                    values.useCustomTitleRow === 'custom' &&
+                    exampleCustomTitleRow
+                  }
+                  formControls={
+                    <>
+                      <TabularModalPage.FormSecondaryButton
+                        onClick={() => {
+                          setIsOpen(false);
+                        }}
+                      />
+                      <TabularModalPage.FormPrimaryButton
+                        onClick={() => undefined}
+                        isDisabled
+                      />
+                      <TabularModalPage.FormDeleteButton
+                        onClick={() => undefined}
+                        isDisabled
+                      />
+                    </>
+                  }
+                  hideControls={values.hideControls}
+                >
+                  <Switch>
+                    <Route
+                      path={[
+                        '/custom-applications/playground/tabular-modal-page/', // production
+                        '/tabular-modal-page/', // development
+                      ]}
```

Comment:
Instead of this, I think we should be able to use Gatsby's `withPrefix` function. https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/#add-the-path-prefix-to-paths-using-withprefix

```suggestion
                      path={withPrefix('/tabular-modal-page/')}
```